config/i18n-tasks.yml

```diff
@@ -106,16 +106,34 @@ ignore_missing:
     - 'deed.deed.*'
     - 'article.graph.title'
     - 'thredded.*'
-  es: &NOT_EN
```

Comment:
aliases in yml files are now considered unsafe and not supported anymore.

Reply:
Reason: aliases in yml files are unsafe and rails moved away from supporting this